modules/core/test/v2/unit/coins/avaxcToken.ts

```diff
@@ -0,0 +1,68 @@
+import 'should';
```

Comment:
please add `@prettier` annotation and format

Reply:
done